### ISSUE TITLE
32bit OSX package for clementine 1.0.1

### ISSUE THREAD

**Issue by Clementine-Issue-Importer** [clementine-player/Clementine#2783]
_From [hansvsch...@gmail.com](https://code.google.com/u/104138615109374465085/) on March 06, 2012 13:26:38_

As a user of one of the early intel macs, containing a Intel Core Duo processor which isn't capable of 32bit, I would like to see a 32bit build of the OSX version for 1.0.1.
However, when trying to compile it myself, i quickly ran into the problem of only being able to get a 64bit version of the Qt framework for OSX. Is this the reason no 64bit OSX version exists?

My problem was fixed by downloading version 0.7 (which is also awesome), however it wants to update to 1.0.1. the upgrade succeeds without problems, but installs the 64bit version which simply doesn't run on my hardware.

to summarize my request: disable the update when running on 32bit OSX & put the 0.7 version in the top of the download section, or please find a way to build a 32bit OSX 1.0.1 version.

_Original issue: http://code.google.com/p/clementine-player/issues/detail?id=2783_
